### ISSUE TITLE
Conversion: DuckDB Type BLOB -> postgres bytea failed

### ISSUE THREAD

**Issue by bobir01** [duckdb/pg_duckdb#464]
### What happens?

When querying parquet that has blob columns (some engines write strings as blob) postrges can't convert them into `bytea`

### To Reproduce

```
select * from read_parquet('s3://bucket/file.parquet') as (product_id varchar(255));
```
```
select * from read_parquet('s3://bucket/file.parquet') as (product_id bytea);
```
the same issue occured on both convertions
```
WARNING:  (PGDuckDB/GetPostgresDuckDBType) Could not convert DuckDB type: BLOB to Postgres type
ERROR:  (PGDuckDB/CreatePlan) Cache lookup failed for type 0
```

### OS:

Linux aarch64

### pg_duckdb Version (if built from source use commit hash):

0.1.0

### Postgres Version (if built from source use commit hash):

15.10

### Full Name:

Bobirjon Mardonov

### Affiliation:

Korzinka GO

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

Not applicable - the reproduction does not require a data set

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by JelteF** [comment]
Thanks for the report. I think it should be fairly easy to implement support for this type. PRs are welcome.

**Comment by ritwizsinha** [comment]
Able to replicate the issue,  I am assuming, we need to add the type `BYTEAOID` to return when type is blob  and add a conversion method for that?

**Comment by JelteF** [comment]
@ritwizsinha yeah that's basically it. A few more things you didn't explicitely mention:
1. `BYTEAARRAYOID` should also be supported (should not be hard to do)
2. The conversion you mention needs tests

**Comment by ritwizsinha** [comment]
The documentation describes support for binary, but I couldn't find the type for that

**Comment by JelteF** [comment]
@ritwizsinha I think the documentation is simply incorrect in that regard.
